### DISCUSSION TITLE
Positioning mesh children, getting position

### DISCUSSION THREAD

**Discussion by rossmarinaro** [enable3d/enable3d#355] [Help / Questions]
I dont know why this is so hard for me to figure out, but I just want to get the updated world position of an element of my glb mesh, for example the muzzle point of a gun model so I can spawn particles from that exact point, only I cannot find a solution. 

This below example doesn't seem to update the position when the object moves.

Mesh.traverse(i=>{
   if (i === 'muzzle')
const pos = i.getWorldPosition() 
console.log(pos)
})

**Comment by yandeu** [comment]
I'm sorry, I can't tell what's wrong off top of my head.
I guess this is more a question for the three.js forum?

**Reply by rossmarinaro** [reply]
I just made some boxes in blender and positioned them in front of the rifle that way. I haven't found a solution in threejs.
